### PR TITLE
feat: traceloop docs

### PR DESCRIPTION
**Pull request by nirga**
Added docs on how to instrument Traceloop to monitor the quality of an LLM app.

### CODE REVIEW

**Comment by ghost** [comment]
hey @nirga - can you explain how this works with litellm? 

I'm confused

**Comment by ishaan-jaff** [comment]
Can we add this as an integration? It would need to get added under the integrations dir

**Comment by ishaan-jaff** [comment]
this is an example observability integration PR: https://github.com/BerriAI/litellm/pull/181 
cc @nirga let me know if it makes sense to integrate traceloop in this way ?

**Comment by nirga** [comment]
Done! @ishaan-jaff WDYT?

File: litellm/integrations/traceloop.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import sys
+from traceloop.tracing import Tracer
+
+
+class TraceloopLogger:
+    def __init__(self):
+        self.tracer = Tracer(sys.argv[0])
```
Comment (ishaan-jaff) [inline litellm/integrations/traceloop.py]: do you not need a `def log_event` in traceloop.py ?

**Review by ishaan-jaff — COMMENTED** [review]
lgtm ! just one question left in the comments

File: litellm/integrations/traceloop.py (hunk 1 above)
Comment (nirga) [reply litellm/integrations/traceloop.py]: Nope @ishaan-jaff :)
We're monkey-patching all the calls and log everything asynchronously with OpenTelemetry.

**Comment by ghost** [comment]
Hey @nirga, 

Can you run 3 tests and attach screenshots to confirm this works : 
```
# Test 1: On completion call
response = completion(model="claude-instant-1", messages=[{"role": "user", "content": "Hi 👋 - i'm openai"}])
# print(f"response: {response}")

# # Test 2: On embedding call
# response = embedding(model="text-embedding-ada-002", input=["sample text"])
# print(f"response: {response}")

# # Test 3: On streaming completion call
response = completion(model="replicate/llama-2-70b-chat:58d078176e02c219e11eb4da5a02a7830a283b14cf8f94537af893ccff5ee781", messages=[{"role": "user", "content": "Hi 👋 - i'm openai"}], stream=True)
print(f"response: {response}")
```

**Comment by ishaan-jaff** [comment]
lgtm ! should be good to merge after addressing @krrishdholakia's request above !

**Comment by nirga** [comment]
Done!
Screenshot 2023-08-29 at 0 20 50
Screenshot 2023-08-29 at 0 21 04
Screenshot 2023-08-29 at 0 22 14

**Comment by ghost** [comment]
Great - will merge by EOD, thanks!

**Pull request merged**